SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/GridBasedLBStrategy.h

```diff
@@ -7,6 +7,7 @@
 #include "CoreMinimal.h"
 #include "Math/Box2D.h"
 #include "Math/Vector2D.h"
+#include "Schema/Interest.h"
```

Comment:
chuck this on line 6 pls, prefer to have GDK includes before engine includes

Reply:
also don't think it's needed given the forward declaration in the abstract lb header file?

Reply:
removed